CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+
+The is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
```

Comment:
How can a major/X release have nothing user facing?  Probably worth at least briefly mentioning the breaking changes to tk-jetty9's API.